ngi_reports/utils/statusdb.py

```diff
@@ -52,7 +68,9 @@ def get_entry(self, name, use_id_view=False):
             return None
         return self.db.get(view.get(name))
```

Comment:
I'm not sure how to handle this for `NanoporeRunConnection`, any ideas?

Reply:
```suggestion
        if self.db:
           doc = self.db.get(view.get(name))
        elif self.dbname:
            doc = self.cloudant.get_document(db=self.dbname, doc_id=view.get(name)).get_result()
        return doc
```
maybe? I haven't tested it